apps/web/components/LanguageSelector/flags.ts

```diff
@@ -0,0 +1,10 @@
+import { US } from 'country-flag-icons/string/3x2';
+import { DE } from 'country-flag-icons/string/3x2';
+
+export interface FlagImports {
+  [key: string]: string;
+}
+export const flagImports: FlagImports = {
+  en: US as string,
+  de: DE as string,
+};
```

Comment:
Cant we import these flags in the nuxt.config.ts?
e.g.:

`[
      '@nuxtjs/i18n',
      {
        locales: [
          {
            code: 'en',
            file: 'en.json',
            flag: abcdef,
          },
          {
            code: 'de',
            file: 'de.json',
            flag: abcdef,
          },
        ],
        lazy: true,
        langDir: 'lang',
        defaultLocale: 'en',
      },
    ],`

Reply:
tried that, vite/nuxt need specific import from a while when dinamically importing